### PR TITLE
Adding some Templates & MIT License

### PR DESCRIPTION
**Pull request by hasezoey**
- adding Templates
- adding MIT License (as Specified in package.json)

i think the following labels must be created for this pull request:
- docs
- feature
- parity

Please give feedback on what could be added (or removed) from these templates
(and please say if i got any writing mistakes, thanks)

Note: I force push & rebase to keep it only one commit

### CODE REVIEW

**Comment by hasezoey** [comment]
@coveralls what, why, i didn't change anything on the code ...

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# How to Contribue
+
+## Before making a Pull Request
+
+- Make sure every Test passes locally
```
Comment (Ben305) [inline .github/CONTRIBUTING.md]: Make sure all tests pass locally

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- Make sure you have run tslint & if something should come up, fix it to keep the code in one style
+- When you add Documentation, please make sure that it fits with the style
```
Comment (Ben305) [inline .github/CONTRIBUTING.md]: When you add documentation please make sure to keep the existing style

File: .github/ISSUE_TEMPLATE/feature parity.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+*please remove the parts in "---"*
+
+## What to include in your Questions
+
+- remove "not needed" from your parity below
+- Make sure you read [Mastering-Markdown](https://guides.github.com/features/mastering-markdown/) , thanks
+
+---
+
+## Wich Feature
```
Comment (Ben305) [inline .github/ISSUE_TEMPLATE/feature parity.md]: Which feature

File: .github/PULL_REQUEST_TEMPLATE/Feature.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+name: Feature
+about: Create an Feature
```
Comment (Ben305) [inline .github/PULL_REQUEST_TEMPLATE/Feature.md]: Add a feature

File: .github/PULL_REQUEST_TEMPLATE/Docs.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+name: Documentation
+about: Update Documentation
```
Comment (Ben305) [inline .github/PULL_REQUEST_TEMPLATE/Docs.md]: Update documentation

File: .github/ISSUE_TEMPLATE/question.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+name: Question
+about: Ask a Question
```
Comment (Ben305) [inline .github/ISSUE_TEMPLATE/question.md]: Ask a question

File: .github/ISSUE_TEMPLATE/feature parity.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+---
+name: Feature Parity
+about: Ask for Feature Parity with Mongoose
```
Comment (Ben305) [inline .github/ISSUE_TEMPLATE/feature parity.md]: Ask for feature parity with mongoose

File: .github/ISSUE_TEMPLATE/feature parity.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+---
+name: Feature Parity
```
Comment (Ben305) [inline .github/ISSUE_TEMPLATE/feature parity.md]: Feature parity

**Comment by Ben305** [comment]
@hasezoey Thanks, I made some comments (only typos)

**Comment by hasezoey** [comment]
@Ben305 Updated with all your suggestions (still one commit)

**Pull request merged**